src/forms/TextInput.jsx

```diff
@@ -69,6 +71,17 @@ export const TextInput = (props) => {
 	const inputStyles = iconShape && {
 		paddingLeft: `${paddingSize}px`
 	};
+	const customValidityMessage = isValid ? '' : validityMessage;
+
+	let textInput = null;
```

Comment:
why are we setting this to `null` here?

Reply:
Because I was getting a lint message when I just tried to set it to `let textInput`

Reply:
You _should_ be able to just declare the variable, `let textInput;`. I'm not seeing any lint errors when I try it locally. This is just a style nitpick though; not a blocker.

Reply:
Updated :)